mlos_bench/mlos_bench/tunables/tunable.py

```diff
@@ -9,14 +9,29 @@
 import collections
 import logging
 
-from typing import Any, Dict, List, Literal, Optional, Sequence, Tuple, Type, TypedDict, Union
+from typing import Any, Dict, Iterable, List, Literal, Optional, Sequence, Tuple, Type, TypedDict, Union
+
+import numpy as np
 
 _LOG = logging.getLogger(__name__)
 
 
 """A tunable parameter value type alias."""
 TunableValue = Union[int, float, Optional[str]]
 
+"""Tunable value type."""
+TunableValueType = Union[Type[int], Type[float], Type[str]]
+
+"""
+Tunable value type tuple.
+For checking with isinstance()
+"""
+TunableValueTypeTuple = (int, float, str, type(None))
+
+"""The string name of a tunable value type."""
+TunableValueTypeName = Literal["int", "float", "categorical"]
```

Comment:
let's make these type-related changes a separate PR. I'll be happy to merge it before the main one

Reply:
#696